catalog/core/catalog-core-commons/src/main/java/ddf/catalog/filter/delegate/FilterToTextDelegate.java

```diff
@@ -20,6 +20,22 @@
 
 public class FilterToTextDelegate extends FilterDelegate<String> {
 
+    // Custom functions
+    @Override
+    public String propertyIsEqualTo(String functionName, List<Object> arguments, Object literal) {
+        switch (functionName) {
+        case "divisibleBy":
```

Comment:
Well we get it as a String from the FilterFunction I will at least change it to reference DisibleByFunction.FUNCTION_NAME.  They are sort of dynamic in the sense that a new FunctionFactory could be added to support a new function but that being said its an internal API so it wouldn't be an external implementor.